lib/services/sites.test.js

```diff
@@ -28,6 +28,15 @@ describe(_.startCase(filename), function () {
         prefix: 'd/e',
         assetDir: 'public',
         assetPath: '/e'
+      },
+      c: {
+        dir: 'z/c',
+        slug: 'c',
+        host: 'h',
+        path: '/i/j',
```

Comment:
does this test always assume a path exists? can we write a test for root sites that don't have paths?